_inc/client/at-a-glance/style.scss

```diff
@@ -360,6 +360,11 @@
 
 .jp-connection-settings__modal.dops-modal {
 	max-width: 635px;
+  	min-height: 400px;
```

Comment:
You'd need to switch to tabs instead of spaces here and below, otherwise the next person making changes to that file will automatically modify those lines even if they don't want to.